develop/reference_workarounds.rst

```diff
@@ -5,20 +5,23 @@ SQL Support and Workarounds
 
 As Citus provides distributed functionality by extending PostgreSQL, it is compatible with PostgreSQL constructs. This means that users can use the tools and features that come with the rich and extensible PostgreSQL ecosystem for distributed tables created with Citus.
 
-Citus supports all SQL queries on distributed tables, with only these exceptions:
+Citus supports all SQL queries on distributed tables, with a few exceptions.
 
-* Correlated subqueries
-* `Recursive <https://www.postgresql.org/docs/current/static/queries-with.html#idm46428713247840>`_/`modifying <https://www.postgresql.org/docs/current/static/queries-with.html#QUERIES-WITH-MODIFYING>`_ CTEs
+Not yet supported:
+
+* `Recursive CTEs <https://www.postgresql.org/docs/current/static/queries-with.html#idm46428713247840>`_
 * `TABLESAMPLE <https://www.postgresql.org/docs/current/static/sql-select.html#SQL-FROM>`_
```

Comment:
Ops, I forgot to mention. We also support `TABLESAMPLE` for router queries, see https://github.com/citusdata/citus/pull/2332. So, it should also be moved down to `Partially supported section`